### ISSUE TITLE
Featured image in about page

### ISSUE THREAD

**Issue by phungai** [CaiJimmy/hugo-theme-stack#67]
Is there a way to have a featured image on the about page? I've tried adding

> image = "image.jpg"

in about.md and placing the image file in various places (in the page directory, in a new sub-directory of "page" called "about" with about.md, and in assets/img). It doesn't seem to work in my case:

![about](https://user-images.githubusercontent.com/75188140/102090297-e9d6be00-3de2-11eb-80d7-e5ce0062af50.png)

I'm using Hugo extended version 0.79.0 on Windows 7.

**Comment by CaiJimmy** [comment]
Create a `about` folder inside `page` directory, with the following structure:

```
├───about
│       image.jpg
│       index.md
```

Markdown file must be named as `index.md`, and not `about.md`

**Comment by phungai** [comment]
Thanks for your prompt response!

That solution works, but the about page shows up in the archive page even when I add

> hidden: true

in the front matter of the index.md file for the about page. I also modified the theme to use MathJax instead of KaTeX by roughly following the instructions [here](https://geoffruddock.com/math-typesetting-in-hugo/), so its possible I broke something with my customization. I may play around with your solution again someday, but for now I'll just create an about page in the standard way.
